java-agent/instrumentation/reactor-core-3.1/src/main/java8/io/opentelemetry/auto/instrumentation/reactor/core/ReactorCoreAdviceUtils.java

```diff
@@ -12,18 +10,23 @@
 import reactor.core.publisher.Mono;
 import reactor.util.context.Context;
 
+import java.util.function.Function;
+
+import static io.opentelemetry.auto.instrumentation.reactor.core.ReactorCoreDecorator.DECORATE;
+import static reactor.core.publisher.Operators.lift;
```

Comment:
Can you check your [Intellij formatting settings](https://github.com/open-telemetry/opentelemetry-auto-instr-java/blob/master/CONTRIBUTING.md#intellij-idea)?